schemas/constants.yml

```diff
@@ -12,6 +12,10 @@ identifier-pattern:     "^([a-zA-Z0-9-_]*)$"
 identifier-min-length:  1
 identifier-max-length:  22
 
+github-installation-pattern:  "^[0-9]+$"
+github-installation-min-length: 1
+github-installation-max-length: 10
```

Comment:
I think we can just count this as an integer, and not try to apply min/max length. So no need for these constants.